src/uint/boxed/cmp.rs

```diff
@@ -10,6 +10,28 @@ use subtle::{
     Choice, ConditionallySelectable, ConstantTimeEq, ConstantTimeGreater, ConstantTimeLess,
 };
 
+impl BoxedUint {
+    /// Returns the Ordering between `self` and `rhs` in variable time.
+    pub fn cmp_vartime(&self, rhs: &Self) -> Ordering {
```

Comment:
I have a feeling that using comparisons between limbs instead of subtraction may be faster. Put a TODO perhaps?